drivers/Aqara/aqara-presence-sensor/src/discovery.lua

```diff
@@ -100,7 +100,7 @@ local function discovery_device(driver)
         break
       end
     end
-    if (not processing_devices[dni]) and (not is_already_added) then
```

Comment:
It seems like this will cause a lot of extra device added messages from the driver to the cloud if we have already started adding the device to the platform, but the hub hasn't gotten the device record from the platform yet. Why is the `processing_devices` not needed anymore? It seems like its purpose is to prevent sending multiply `try_create_device` requests, which seems like something that we want to do.